server/channels/app/import_functions_test.go

```diff
@@ -522,7 +522,8 @@ func TestImportImportTeam(t *testing.T) {
 	require.NoError(t, err, "Failed to get team count.")
 
 	// we also assert that the team name can be upper case
-	teamName := "A" + model.NewId()
+	// Note there are no reserved team names starting with `Z`, making this flake-free.
+	teamName := "Z" + model.NewId()
```

Comment:
Should we use `model.NewRandomTeamName` instead?

Reply:
I think `model.NewRandomTeamName` doesn't generate capital letters?

Reply:
Got it. I'll probably leave the existing semantics for now, but food for thought on `model.NewRandomTeamName` in the future.